http-server-netty/src/main/java/io/micronaut/http/server/netty/types/files/FileTypeHandler.java

```diff
@@ -157,8 +157,13 @@ protected void setDateHeader(MutableHttpResponse response) {
         headers.date(now);
     }
 
-    private FullHttpResponse notModified() {
+    private static void copyHeaders(NettyMutableHttpResponse<?> from, NettyMutableHttpResponse to) {
+        from.getHeaders().forEachValue((header, value) -> to.getHeaders().set(header, value));
```

Comment:
I think `set(..)` will override the header for multi value headers and `add` should be used instead

Reply:
Done.